media/js/base/core-datalayer.js

```diff
@@ -25,6 +25,33 @@ if (typeof Mozilla == 'undefined') {
         return pageId ? pageId : pathName.replace(/\/[^\/]*/, '');
     };
 
+    /** Returns whether page has download button.
+    * @param {String} path - URL path name fallback if page ID does not exist.
+    * @return {String} boolean.
+    */
+    Analytics.pageHasDownload = function() {
+        return ( $('[data-download-os]').length > 0 ) ? 'true' : 'false';
```

Comment:
Also is there any reason you're looking for a data attribute specifically? Technically, looking for a class like `.download-link` would be faster?